SNUTT-2022/SNUTT/Models/Lecture.swift

```diff
@@ -36,6 +36,8 @@ struct Lecture: Identifiable {
     var registrationCount: Int?
     var wasFull: Bool
 
+    var evLecture: EvLecture?
+
```

Comment:
```suggestion
    private(set) var evLecture: EvLecture?
```

나머지 속성도 private(set)으로 변경하거나 mutable일 필요 없는 것들은 let으로 바꿔주면 좋을듯~